.github/scripts/build-zip.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
 set -eo pipefail
 
-if [[ -z "$PACKAGE_VERSION" ]]; then
-	echo "Missing PACKAGE_VERSION env var"
-	exit 1
-fi
-
 PLUGIN_ZIP_FILENAME="elementor-${PACKAGE_VERSION}.zip"
 npx grunt build
 mv build elementor
 zip -r $PLUGIN_ZIP_FILENAME elementor
 echo "PLUGIN_ZIP_FILENAME=${PLUGIN_ZIP_FILENAME}" >> $GITHUB_ENV
+echo "PLUGIN_ZIP_PATH=${WORKSPACE}/elementor/**/*" >> $GITHUB_ENV
```

Comment:
Explain

Reply:
The core & pro are creates the build file in different location which sets within the plugin grunt config.